pkg/reconciler/vsphere/resources/names/names.go

```diff
@@ -22,18 +22,22 @@ import (
 	"github.com/mattmoor/vmware-sources/pkg/apis/sources/v1alpha1"
 )
 
-func DeploymentName(vms *v1alpha1.VSphereSource) string {
-	return kmeta.ChildName(vms.Name, "deployment")
+func Deployment(vms *v1alpha1.VSphereSource) string {
```

Comment:
I think having the Name in the method makes it clearer to the reader where this code is being used.

Reply:
Go style generally doesn't like the redundancy with the package name since you get a "stutter": `names.DeploymentName` (name it there twice).